Story/LordsofChaos/Core13LoC.cs

```diff
@@ -1923,7 +1923,7 @@ public void Xiang()
         }
 
         //Pure Chaos
-        if (!Story.QuestProgression(3185) || (Core.isCompletedBefore(3188) ? false : !Core.CheckInventory("nchaorrupted Sample")) )
+        if (!Story.QuestProgression(3185) || (Core.isCompletedBefore(3188) ? false : !Core.CheckInventory("Unchaorrupted Sample")) )
```

Comment:
another typo.. 🤣  ty

Reply:
Someone reported it in discord 💀I think that’s the issue bot failed to try quests below it